include/vptr/virtual_ptr.hpp

```diff
@@ -30,6 +30,9 @@
  *
  **************************************************************************/
 
+#ifndef VIRTUAL_PTR_HPP
```

Comment:
mm, that's not a bad idea. We're currently using the sdk files through a submodule, so we shouldn't come across the problem, but I image that it could be an issue for other projects.

Reply:
I've made the header guard more complex, which should hopefully not clash with anything else.